symbolic-il2cpp/src/usym.rs

```diff
@@ -170,26 +168,50 @@ mod raw {
     }
 }
 
-/// A record mapping an IL2CPP instruction address to managed code location.
-///
-/// Not all native code maps back to managed code, for those records the managed info will
-/// be `None`.
-#[derive(Debug, Clone)]
-pub struct UsymSourceRecord<'a> {
+#[derive(Clone)]
+pub struct UnmappedRecord<'a> {
+    /// Instruction pointer address, relative to the base of the assembly.
+    pub address: u64,
+    /// Symbol name of the native code. Kept as bytes for lossless comparison.
+    pub native_symbol_bytes: &'a [u8],
```

Comment:
A combination of a record's native file path and symbol are used to construct a temporary identifier to match up records that belong to the same function. As mentioned in the documentation, these are kept as bytes for lossless comparison, but I would be open to the idea of simply converting the symbols and fileinfos to `Cow`s instead.